fixture/plugin.cpp

```diff
@@ -0,0 +1,161 @@
+#include "plugin.h"
```

Comment:
Same deal, totally stripped down plugin with 0 logic, just an interface to test the wrapper against.